### PR TITLE
Sync turntables and transfertables in multiplayer mode

### PR DESCRIPTION
**Pull request by Csantucci**
https://blueprints.launchpad.net/or/+spec/sync-turntables-in-multiplayer
http://www.elvastower.com/forums/index.php?/topic/28591-operational-turntable/page__view__findpost__p__287727

### CODE REVIEW

File: Source/Orts.Simulation/MultiPlayer/Message.cs
Comment (Sharpe49) [inline Source/Orts.Simulation/MultiPlayer/Message.cs:3907]: The name of public members should start with an upper case character.

File: Source/Orts.Simulation/MultiPlayer/Message.cs
Comment (Sharpe49) [inline Source/Orts.Simulation/MultiPlayer/Message.cs:2784]: Why is this being indented?

File: Source/Orts.Simulation/MultiPlayer/Message.cs
Comment (Sharpe49) [inline Source/Orts.Simulation/MultiPlayer/Message.cs:2350]: Why is this being indented?

File: Source/Orts.Simulation/Simulation/Transfertables.cs
Comment (Sharpe49) [inline Source/Orts.Simulation/Simulation/Transfertables.cs:194]: Maybe, put this formula in a getter (at line 50)

File: Source/Orts.Simulation/Simulation/Turntables.cs
Comment (Sharpe49) [inline Source/Orts.Simulation/Simulation/Turntables.cs:84]: This should be in CamelCase.

**Comment by Csantucci** [comment]
Code cleaned up as requested

File: Source/Orts.Simulation/Simulation/Turntables.cs
Comment (Sharpe49) [reply Source/Orts.Simulation/Simulation/Turntables.cs]: Since this an enum type definition, this should be UpperCamelCase, not lowerCamelCase.

Comment (Csantucci) [reply Source/Orts.Simulation/Simulation/Turntables.cs]: > Since this an enum type definition, this should be UpperCamelCase, not lowerCamelCase.

Done.

**Review by Sharpe49 — APPROVED** [review]

**Comment by Csantucci** [comment]
Thanks. Waiting for the approval of the blueprint before merging.

**Comment by cjakeman** [comment]
Hi Carlo,

> Thanks. Waiting for the approval of the blueprint before merging.

Blueprint approved, but the PR now has a conflict.

**Comment by Csantucci** [comment]
Thanks Chris. Conflict removed.
Thaks Cédric for immediate renewed approval.

**Comment by Csantucci** [comment]
Waiting a couple of days before merging.

**Pull request merged**
